portfolio/src/main/webapp/index.html

```diff
@@ -8,11 +8,45 @@
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+      <div id="galleryDiv">
+        <h1>Gallery</h1>
+        <div class="gallery">
+          <a target="_blank" href="images/remarkables.jpg">
+            <img src="images/remarkables.jpg" width="600" height="400">
```

Comment:
As Lilian's previous advice for me, it is better to split width and height out as separate style requirement for img in "style.css":
.img{width = ...; ...}

Reply:
ok will do. Thanks!